docs/admin-manual/configuration-templates.rst

```diff
@@ -315,6 +315,23 @@ incorporates.
        ``HOLD_REASON_CPU_EXCEEDED`` and ``HOLD_SUBCODE_CPU_EXCEEDED``
        respectively.
 
+    -  ``Preempt_If_Disk_Exceeded``
+
+       If the startd observes the amount of disk space used by the job
+       exceed the disk in the slot, preempt the job immediately
+       ignoring any job retirement time.
+
+    -  ``Hold_If_Cpus_Exceeded``
```

Comment:
This should be Hold_If_Disk_Exceeded, right?

Reply:
indeed